src/Microsoft.ML.FastTree/GamClassification.cs

```diff
@@ -157,14 +156,17 @@ protected override SchemaShape.Column[] GetOutputColumnsCore(SchemaShape inputSc
         }
     }
 
-    public class BinaryClassificationGamModelParameters : GamModelParametersBase, IPredictorProducing<float>
+    /// <summary>
+    /// The model parameters class for Binary Classification GAMs
+    /// </summary>
+    public sealed class BinaryClassificationGamModelParameters : GamModelParametersBase, IPredictorProducing<float>
     {
         internal const string LoaderSignature = "BinaryClassGamPredictor";
         public override PredictionKind PredictionKind => PredictionKind.BinaryClassification;
 
-        internal BinaryClassificationGamModelParameters(IHostEnvironment env, int inputLength, Dataset trainset,
-            double meanEffect, double[][] binEffects, int[] featureMap)
-            : base(env, LoaderSignature, inputLength, trainset, meanEffect, binEffects, featureMap) { }
+        public BinaryClassificationGamModelParameters(IHostEnvironment env,
```

Comment:
>BinaryClassificationGamModelParameters [](start = 15, length = 38)

Since you are making this public, could you also add XML documentation of what it does and what the parameters are?

Reply:
Done.